themes/bootstrap3/templates/search/backtoorigin.phtml

```diff
@@ -0,0 +1,11 @@
+<?php
+/** @var \Laminas\View\Renderer\PhpRenderer $this */
+/** @var \VuFind\Search\SearchOrigin\AbstractSearchOrigin|null $searchOrigin */
+if (isset($searchOrigin)) {
```

Comment:
`$searchOrigin` should have a name that implies it's specific to browse search, or alternatively the class should contain a search type indicator that should be checked here. Otherwise the translation key `back_to_browse_by` doesn't quite fit into this all.